upgrade/1.4.4/CSM-Only.md

```diff
@@ -0,0 +1,138 @@
+
+   This script creates a new CFS configuration that includes the CSM version in its name and applies it to the
+   management nodes. This leaves the management node components in CFS disabled. They will be automatically enabled when
+   they are rebooted at a later stage in the upgrade.
+
+   ```bash
+   /usr/share/doc/csm/scripts/operations/configuration/apply_csm_configuration.sh \
+       --no-enable --config-name "management-${CSM_RELEASE}"
+   ```
+
+   Successful output should end with the following line:
+
+   ```text
+   All components updated successfully.
+   ```
+
+1. (`ncn-m001#`) Set the [IMS](../../glossary.md#image-management-service-ims) IDs for each Management node sub-role.
+
+    * Kubernetes ID
+
+      ```bash
+      KUBERNETES_IMAGE_ID="$(kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' \
+        | yq r -j - '"'${CSM_RELEASE}'".images' \
+        | jq -r '. as $o | keys_unsorted[] | select(startswith("secure-kubernetes")) | $o[.].id')"
+      export KUBERNETES_IMAGE_ID
+      echo "KUBERNETES_IMAGE_ID=$KUBERNETES_IMAGE_ID"
+      ```
+
+    * Storage-CEPH ID
+
+      ```bash
+      STORAGE_IMAGE_ID="$(kubectl -n services get cm cray-product-catalog -o jsonpath='{.data.csm}' \
+        | yq r -j - '"'${CSM_RELEASE}'".images' \
+        | jq -r '. as $o | keys_unsorted[] | select(startswith("secure-storage")) | $o[.].id')"
+      export STORAGE_IMAGE_ID
+      echo "STORAGE_IMAGE_ID=$STORAGE_IMAGE_ID"
+      ```
+
+1. (`ncn-m001#`) Create CFS sessions for both Kubernetes and Storage nodes.
+
+    1. Build customized master node images.
+
+       ```bash
+       cray cfs sessions create \
+           --target-group Management_Master \
+           "$KUBERNETES_IMAGE_ID" \
+           --target-definition image \
+           --target-image-map "$KUBERNETES_IMAGE_ID" "management-kubernetes-${CSM_RELEASE}" \
+           --configuration-name "management-${CSM_RELEASE}" \
+           --name "management-kubernetes-${CSM_RELEASE}-upgrade" \
+           --format json
+       ```
+
+    1. Build customized storage node images.
+
+       ```bash
+       cray cfs sessions create \
+           --target-group Management_Storage \
+           "$STORAGE_IMAGE_ID" \
+           --target-definition image \
+           --target-image-map "$STORAGE_IMAGE_ID" "management-storage-${CSM_RELEASE}" \
+           --configuration-name "management-${CSM_RELEASE}" \
+           --name "management-storage-${CSM_RELEASE}-upgrade" \
+           --format json
+       ```
+
+1. (`ncn-m001#`) Wait for the image builds to complete successfully.
+
+   > This may take over 20 minutes to complete.
+
+   ```bash
+   watch '
+   cray cfs sessions describe "management-kubernetes-${CSM_RELEASE}-upgrade" --format json | jq -r ".status.session.succeeded"
+   cray cfs sessions describe "management-storage-${CSM_RELEASE}-upgrade" --format json | jq -r ".status.session.succeeded"
```

Comment:
```suggestion
    1. Set variables with unique names for the CFS sessions to be created.

       ```bash
       TSTAMP=$(date "+%Y%m%d%H%M%S")
       K8S_CFS_SESSION_NAME="management-kubernetes-${CSM_RELEASE}-${TSTAMP}"
       STOR_CFS_SESSION_NAME="management-storage-${CSM_RELEASE}-${TSTAMP}"
       ```

    1. Build customized master node images.

       ```bash
       cray cfs sessions create \
           --target-group Management_Master \
           "$KUBERNETES_IMAGE_ID" \
           --target-definition image \
           --target-image-map "$KUBERNETES_IMAGE_ID" "management-kubernetes-${CSM_RELEASE}" \
           --configuration-name "management-${CSM_RELEASE}" \
           --name "${K8S_CFS_SESSION_NAME}" \
           --format json
       ```

    1. Build customized storage node images.

       ```bash
       cray cfs sessions create \
           --target-group Management_Storage \
           "$STORAGE_IMAGE_ID" \
           --target-definition image \
           --target-image-map "$STORAGE_IMAGE_ID" "management-storage-${CSM_RELEASE}" \
           --configuration-name "management-${CSM_RELEASE}" \
           --name "${STOR_CFS_SESSION_NAME}" \
           --format json
       ```

1. (`ncn-m001#`) Wait for the image builds to complete successfully.

   > This may take over 20 minutes to complete.

   ```bash
   watch '
   cray cfs sessions describe "${K8S_CFS_SESSION_NAME}" --format json | jq -r ".status.session.succeeded"
   cray cfs sessions describe "${STOR_CFS_SESSION_NAME}" --format json | jq -r ".status.session.succeeded"
```